extern/algebra-plugins/CMakeLists.txt

```diff
@@ -5,45 +5,46 @@
 # Mozilla Public License Version 2.0
 
 # CMake include(s).
-cmake_minimum_required( VERSION 3.14 )
-include( FetchContent )
+cmake_minimum_required(VERSION 3.14)
+include(FetchContent)
 
 # Tell the user what's happening.
-message( STATUS "Building Algebra Plugins as part of the Detray project" )
+message(STATUS "Building Algebra Plugins as part of the Detray project")
 
 # Declare where to get Algebra Plugins from.
-set( DETRAY_ALGEBRA_PLUGINS_SOURCE
-   "URL;https://github.com/acts-project/algebra-plugins/archive/refs/tags/v0.13.0.tar.gz;URL_MD5;dca48f56be3f8d9526ddf0fbb6fe672b"
-   CACHE STRING "Source for Algebra Plugins, when built as part of this project" )
-mark_as_advanced( DETRAY_ALGEBRA_PLUGINS_SOURCE )
-FetchContent_Declare( AlgebraPlugins ${DETRAY_ALGEBRA_PLUGINS_SOURCE} )
+set(DETRAY_ALGEBRA_PLUGINS_SOURCE
+   "GIT_REPOSITORY;https://github.com/beomki-yeo/algebra-plugins;GIT_TAG;jacobian-transformation"   
```

Comment:
This gets changed after the rollback in algebra-plugins, I trust?

Reply:
Yes